addons/binding/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/handler/IPBridgeHandler.java

```diff
@@ -55,6 +55,8 @@
 
     private static final Integer SYSTEM_DBEXPORTDATETIME = 10;
 
+    private static final Integer MAX_LOGIN_ATTEMPTS = 2;
```

Comment:
I've been out of the Java game for a while, so I'm not sure I understand your comment regarding autoboxing. I just made it the same type as `SYSTEM_DBEXPORTDATETIME` to be consistet. I'll change it to an `int`.